inference-engine/tests/functional/plugin/plaidml/shared_tests_instances/single_layer_tests/embedding_bag_offsets_sum.cpp

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+
+#include "single_layer_tests/embedding_bag_offsets_sum.hpp"
+#include "common_test_utils/test_constants.hpp"
+
+using namespace LayerTestsDefinitions;
+
+namespace {
+
+const std::vector<InferenceEngine::Precision> netPrecisions = {
+        InferenceEngine::Precision::FP32,
+        InferenceEngine::Precision::I32,
+};
+
+const std::vector<InferenceEngine::Precision> indPrecisions = {
+        InferenceEngine::Precision::I32
+};
+
+const std::vector<std::vector<size_t>> embTableShape = {{5, 6}, {10, 35}, {5, 4, 16}};
+const std::vector<std::vector<size_t>> indices =
+        {{0, 1, 2, 2, 3}, {4, 4, 3, 1, 0}, {1, 2, 1, 2, 1, 2, 1, 2, 1, 2}};
+const std::vector<std::vector<size_t>> offsets = {{0, 2}, {0, 0, 2, 2}, {2, 4}};
+const std::vector<size_t> defaultIndex = {0, 4};
+const std::vector<bool> withWeights = {false};
```

Comment:
I'm not sure how [EmbeddingBagOffsetsSumLayerTest::SetUp](https://github.com/Flex-plaidml-team/openvino/blob/zhibinli-embedding_bag_offsets_sum/inference-engine/tests/functional/plugin/shared/src/single_layer_tests/embedding_bag_offsets_sum.cpp) processes this when `withWeights = true` because the setup doesn't read in `per_sample_weights` as the docs says at all. Even the tests under CPU folder didn't pass in `per_sample_weights`. Any info?

Reply:
It looks to me like in this case, the weights are not provided in the shared_tests_instance but rather are produced in `makeEmbeddingBagOffsetsSum`. Specifically, in `inference-engine/tests/ngraph_functions/src/embedding_bag_offsets_sum.cpp` there is a call to `makeConstant` in the `withWeights == true` case. This will produce a pseudo-random (with fixed seed) constant for these weights, see the definition of `makeConstant` in `inference-engine/tests/ngraph_functions/include/ngraph_functions/builders.hpp`.